ui/admin/tests/acceptance/credential-store/delete-test.js

```diff
@@ -96,9 +102,7 @@ module('Acceptance | credential-stores | delete', function (hooks) {
         (item) => item !== 'delete',
       );
     await visit(urls.vaultCredentialStore);
-    assert.notOk(
-      find('.rose-layout-page-actions .rose-dropdown-button-danger'),
-    );
+    assert.dom(DELETE_ACTION_SELECTOR).doesNotExist();
```

Comment:
```suggestion
    await click(MANAGE_DROPDOWN_SELECTOR);
    
    assert.dom(DELETE_ACTION_SELECTOR).doesNotExist();
```
i think we need to click the dropdown first to make sure the action is not there since only the link is wrapped in the `#if` check.

Reply:
Good catch. Thanks Lis!

Reply:
Fixed.